pkg/scheduler/controller/shoot/reconciler.go

```diff
@@ -299,6 +303,21 @@ func filterSeedsForAccessRestrictions(seedList []gardencorev1beta1.Seed, shoot *
 	return seedsSupportingAccessRestrictions, nil
 }
 
+// filterSeedsWithDisabledShootReconciliations filters seeds which have annotation set to temporarily disable shoot reconciliations.
+func filterSeedsWithDisabledShootReconciliations(seedList []gardencorev1beta1.Seed) ([]gardencorev1beta1.Seed, error) {
```

Comment:
How is the `gardener-scheduler` retriggered for scheduling pending `Shoot`s once the emergency annotation was removed?